lib/cmds/space_cmds/create.js

```diff
@@ -33,6 +35,11 @@ export const builder = (yargs) => {
       alias: 'org',
       describe: 'Organization owning the new space'
     })
+    .option('yes', {
+      alias: 'y',
```

Comment:
what do you think about `--force` `-f` instead?

Reply:
I like `--force` better than `--yes` 👍

Reply:
`--yes` is a Linux standardized thing for yes or no questions and we are doing this for other commands

Reply:
ah okay, didn't realize. then sure, we keep the `--yes` :)